src/configs/sastconfig.c4m

```diff
@@ -26,29 +26,38 @@ You can configure the following fields in the tool.semgrep object:
 
 semgrep_config_profile: The semgrep profile to use.  Defaults to 'auto'
 semgrep_format:         The output format flag to pass. Defaults to 'sarif'
-semgrep_metrics:        Whether to ping semgrep. 'on' or 'off'. Defaults to 'off'
+semgrep_metrics:        Whether to ping semgrep. 'on' or 'off'. Defaults to 'on' to be compatible with config=auto
 """
 }
 
+
+
 func find_semgrep(ignore) {
-  result := find_exe("semgrep", [])
+  semgrep_path := "/tmp/semgrep.env/bin/semgrep"
+  result := find_exe(semgrep_path, [])
```

Comment:
find_exe() IIRC searches the $PATH too, even if you call it with a full path. I don't know why we'd prioritize looking in /tmp if it's already installed on the system??  

I guess if we don't like the version installed?

Reply:
mixed up the argument order here which should be fixed now, but yes we want to ensure we have the right version